### ISSUE TITLE
working group communication channels

### ISSUE THREAD

**Issue by mkuzak** [SoftDev4Research/working-group#1]
Shall we use this repositories issues as a general group communication channel?

**Comment by rajido** [comment]
Yes, I think that's a good idea. I think any relevant information for the group should be posted in this working-group repository. Anything specific should probably go to one of the other repos. For instance a query about metrics I think it should go to the metrics repository issue tracking. What do you think? We can use the issue tracking system for actions, queries, ideas, ... and we can organised actions in milestones.

**Comment by mkuzak** [comment]
This sounds good. I'll also setup gitter for chat, which is to much noise for issues.
It will not hurt, in worst case we will not use it.

**Comment by rajido** [comment]
Yes, go ahead and create a channel for more informal communication. I hope important things will end up in the issue tracker which is much easier to structure and follow. Please consider https://fleep.io/ as alternative to Gitter. Fleep can highlight important points in the conversation so it is easier to come back and follow. It can also identify tasks within the conversation. Gitter might be better integrated with Github. I think you have more experience, so please go ahead and create what you think is best.

**Comment by mkuzak** [comment]
Yes, you mentioned fleep yesterday. We can try it out.
